### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing the listitems present under comboxes in "Run on web App for Containers" dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1469]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Create a new java project.
3. Tab to and select Project ->Azure->Push Image.
4. In "Push Image" dialog , tab to container registry and Artifact combo boxes, and use alt+down arrow to expand the combo box
5. Observe the behaviour of Jaws and NVDA

**Actual Result:**
List items under combo boxes are not announced by Jaws and NVDA

**Jaws behaviour:**
Combo box closed
blank
1 of 3
To change the selection use the arrow keys
Open list Box
open
blank
blank
blank

**NVDA behaviour:**
combo box  collapsed
list
list item  selected  1 of 3
list item  selected  2 of 3
list item  selected  3 of 3

**Expected Result:**
List items under combo box should be announced by Jaws and NVDA

Jaws should announce as
Artifacts Combo box closed
1 of 3
To change the selection use the arrow keys
Open list Box
open
unnamed list item  selected  1 of 3
unnamed2 list item  selected  2 of 3
unnamed3 list item  selected  3 of 3

NVDA should announce as
Artifacts combo box  collapsed
list
unnamed list item  selected  1 of 3
unnamed2 list item  selected  2 of 3
unnamed3 list item  selected  3 of 3

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![push image-list items](https://user-images.githubusercontent.com/22956130/36102784-65b42156-1033-11e8-8474-2f76d6ced945.jpg)
![4923159d](https://user-images.githubusercontent.com/22956130/36102788-693344a6-1033-11e8-8ece-922d74fec398.jpg)
![push image-list items-explorer](https://user-images.githubusercontent.com/22956130/36102791-6b91a9fe-1033-11e8-9bc9-1c7ecb4ece76.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749923
